best-practices/haproxy-best-practices.md

```diff
@@ -175,7 +175,7 @@ listen admin_stats                         # The name of the Stats page reportin
 listen tidb-cluster                        # Database load balancing.
    bind 0.0.0.0:3390                       # The Floating IP address and listening port.
    mode tcp                                # HAProxy uses layer 4, the transport layer.
-   balance leastconn                       # The server with the smallest number of connections receives the connection. `leastconn' is recommended where long sessions are expected, such as LDAP, SQL and TSE, rather than protocols using short sessions, such as HTTP. The algorithm is dynamic, which means that server weights might be adjusted on the fly for slow starts for instance.
+   balance leastconn                       # The server with the smallest number of connections receives the connection. `leastconn" is recommended where long sessions are expected, such as LDAP, SQL and TSE, rather than protocols using short sessions, such as HTTP. The algorithm is dynamic, which means that server weights might be adjusted on the fly for slow starts for instance.
```

Comment:
```suggestion
   balance leastconn                       # The server with the smallest number of connections receives the connection. "leastconn" is recommended where long sessions are expected, such as LDAP, SQL and TSE, rather than protocols using short sessions, such as HTTP. The algorithm is dynamic, which means that server weights might be adjusted on the fly for slow starts for instance.
```